changelog/1813.bugfix.rst

```diff
@@ -0,0 +1 @@
+Modified an error message to properly trigger if both `T` and `Vperp` (and `lorentzfactor`) are missing as well, instead of just when there are both of them.
```

Comment:
```suggestion
Modified an error message in `~plasmapy.formulary.lengths.gyroradius` to
properly trigger if both ``T`` and ``Vperp`` (and ``lorentzfactor``) are 
missing as well, instead of just when there are both of them.
```
When single backticks are used, Sphinx tries to create a link to a Python object, like for `gyroradius` above (with the `~` indicating to hide the namespace prefixes). Changing them to double backticks will make it get formatted as literal text (like for a code snippet) without trying to create any links.

The error message that you were getting really is confusing!  I'm going to make a note to create an issue in the Sphinx repository to see if they can improve it.

Reply:
Oh yeah based on the codebase guidance I reversed this change, it should just return nan.